### PR TITLE
Fixes eating grammars

### LINKED ISSUE #39571: texts from eating food seems bugged for most shit
You unwillingly take a inhale of a bit of the wish soup.

### PR DESCRIPTION
**Pull request by kriskog**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Skoglol
fix: Eating is now more gramatically correct
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

Fixes #39571 
"You hungrily take a devour from the dead mouse."

### CODE REVIEW

**Comment by plapatin** [comment]
F

File: code/modules/food_and_drinks/food/snacks.dm [hunk 1]
```diff
@@ -102,13 +102,13 @@ All foods are distributed among various categories. Use common sense.
...
 			else if(fullness <= 50)
-				user.visible_message("<span class='notice'>[user] hungrily takes a [eatverb] from \the [src], gobbling it down!</span>", "<span class='notice'>You hungrily take a [eatverb] from \the [src], gobbling it down!</span>")
+				user.visible_message("<span class='notice'>[user] hungrily [eatverb] \the [src], gobbling it down!</span>", "<span class='notice'>You hungrily [eatverb] \the [src], gobbling it down!</span>")
 			else if(fullness > 50 && fullness < 150)
-				user.visible_message("<span class='notice'>[user] hungrily takes a [eatverb] from \the [src].</span>", "<span class='notice'>You hungrily take a [eatverb] from \the [src].</span>")
+				user.visible_message("<span class='notice'>[user] hungrily [eatverb] \the [src].</span>", "<span class='notice'>You hungrily [eatverb] \the [src].</span>")
 			else if(fullness > 150 && fullness < 500)
-				user.visible_message("<span class='notice'>[user] takes a [eatverb] from \the [src].</span>", "<span class='notice'>You take a [eatverb] from \the [src].</span>")
+				user.visible_message("<span class='notice'>[user] [eatverb] \the [src].</span>", "<span class='notice'>You [eatverb] \the [src].</span>")
```
Comment (ShizCalev) [inline code/modules/food_and_drinks/food/snacks.dm]: [user] slurp the coffee.

Comment (kriskog) [reply code/modules/food_and_drinks/food/snacks.dm]: Yes, you are right. The few that fit before no longer does, I will fix them.

Comment (kriskog) [reply code/modules/food_and_drinks/food/snacks.dm]: Or rather, the observer strings were wrong. Fixed.

**Comment by kriskog** [comment]
Had to remove a few that can't end in 's' for observer.

File: code/modules/food_and_drinks/food/customizables.dm [hunk 2]
```diff
@@ -277,7 +277,7 @@
 
 /obj/item/reagent_containers/food/snacks/customizable/soup/Initialize()
 	. = ..()
-	eatverb = pick("slurp","sip","suck","inhale","drink")
```
Comment (Qustinnus) [inline code/modules/food_and_drinks/food/customizables.dm]: takes a succ

lmao

also
>inhales the soup

what the fuck

Comment (ShizCalev) [reply code/modules/food_and_drinks/food/customizables.dm]: Valid expression actually.

**Review by ninjanomnom — APPROVED** [review]

**Comment by ninjanomnom** [comment]
I'm inhaling soup rn tbh

**Pull request merged**